src/Symfony/Component/Intl/Currencies.php

```diff
@@ -0,0 +1,117 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Intl;
+
+use Symfony\Component\Intl\Exception\MissingResourceException;
+
+/**
+ * Gives access to currency-related ICU data.
+ *
+ * @author Bernhard Schussek <bschussek@gmail.com>
+ * @author Roland Franssen <franssen.roland@gmail.com>
+ */
+final class Currencies extends AbstractStaticBundle
+{
+    private const INDEX_SYMBOL = 0;
+    private const INDEX_NAME = 1;
+    private const INDEX_FRACTION_DIGITS = 0;
+    private const INDEX_ROUNDING_INCREMENT = 1;
+
+    /**
+     * @return string[]
+     */
+    public static function getCodes(): array
```

Comment:
```suggestion
    public static function getCurrencyCodes(): array
```

I'm not a fan of this name. Unlike "ID", "code" is a very ambiguous term that can mean all kinds of things. The *thing* we're talking about is a [currency code](https://www.xe.com/iso4217.php). I would prefer to be clear here (and in the rest of the class).

Reply:
To expand: We have `getNumericCode()` in the same class. There should be a clear distinction in naming between the terms established here.

* currency code -> ISO 3-letter code (commonly called "currency code")
* numeric code -> ISO numeric code

Reply:
what about `Currencies::all()`? That be the SF convention actually :thinking: 

I tend to agree with you, though im still a bit reluctant to change given the classname provides the subject already.

Put different; what about `exists($code)`? Should it be `currencyExists($code)`, `exists($currencyCode)`?